arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -236,11 +248,43 @@
           "nextLinkName": null
         }
       }
+    },
+    "/providers/Microsoft.Sql/operations": {
```

Comment:
They're all under the Microsoft.Sql provider, so calling this actually lists all of them.  I just confirmed this in Fiddler.  Is there a way to define it in one place and reference it from all files?